### DISCUSSION TITLE
How to choose between two kinds of URL(mirror) ?

### DISCUSSION THREAD

**Discussion by SherkeyXD** [termux/termux-packages#10408] [Q&A] [answered]
In wiki, there's two kinds of URL for mirror entry (take main for example)
- https://sitename/termux/termux-packages-24
- https://sitename/termux/apt/termux-main

Which one should I use?

**Comment by Grimler91** [comment, marked as answer]
I would suggest https://{something}/termux-main, most mirrors use that format

**Comment by 2096779623** [comment]
> In wiki, there's two kinds of URL for mirror entry (take main for example)
> 
> * https://sitename/termux/termux-packages-24
> * https://sitename/termux/apt/termux-main
> 
> Which one should I use?

There is one more: https://sitename/termux/termux-main.
